src/commands/deploy.ts

```diff
@@ -82,6 +82,22 @@ export async function deploy(this: IActionContext, target?: vscode.Uri | string
         throw new Error(localize('pythonNotAvailableOnWindows', 'Python projects are not supported on Windows Function apps.  Deploy to a Linux Consumption app.'));
     }
 
+    // we need this check due to this issue: https://github.com/Microsoft/vscode-azurefunctions/issues/625
```

Comment:
Can you move this to it's own function? The 'deploy' function is already too long